integrations/GoogleOptimize/util.js

```diff
@@ -0,0 +1,13 @@
+const ScriptLoaderHead = (id, src, async = defaultAsyncState) => {
+    logger.debug(`in script loader=== ${id}`);
+    const js = document.createElement("script");
+    js.src = src;
+    js.async = async === undefined ? defaultAsyncState : async;
+    js.type = "text/javascript";
+    js.id = id;
+    const e = document.getElementsByTagName("head")[0];
+    logger.debug("==adding script==", js);
+    e.appendChild(js);
```

Comment:
It is always recommended to add the new script to the beginning of the existing ones.

Reply:
`const h = document.getElementsByTagName("head")[0];
  const e = h.getElementsByTagName("script")[0];
  e.parentNode.insertBefore(js, e);`
like this?

Reply:
Yeah, similar to what we do in the common script loader utility

Reply:
Although, you wouldn't need `e.parentNode` rather it should simply be `h.insertBefore(js, e)`

Reply:
updated